test/inspect.test.js

```diff
@@ -1,5 +1,5 @@
 var test = require('tap').test;
-var fs = require('fs');
+//var fs = require('fs');
```

Comment:
let's remove this line

Reply:
It's needed by the test that is temporarily commented out.

Reply:
yep I know, we can restore it once we un-comment the test - it won't run without it :)